bigbluebutton-html5/imports/ui/components/tabs/components/TabContentPdf/TabContentPdf.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import TabPositionButtonGroup from '../TabPositionButtonGroup';
+import Slide from '../Slide';
+
 const TabContentPdf = ({ activeKey, contentKey }) => {
-  const contentShow = activeKey ? 'block' : 'hidden';
+  const contentShow = activeKey === 2 ? 'block' : 'hidden';
```

Comment:
I think this is not good, we relay on activeKey inside the component, instead, we should render the component based on key, not to show all component and then show/hide

Reply:
done